docs/creating-your-site.md

```diff
@@ -62,10 +62,17 @@ theme:
             {
               "yaml.schemas": {
                 "https://squidfunk.github.io/mkdocs-material/schema.json": "mkdocs.yml"
-              }
+              },
+              "yaml.customTags": [
+                "tag:yaml.org,2002:python/name:materialx.emoji.to_svg",
+                "tag:yaml.org,2002:python/name:materialx.emoji.twemoji",
+                "tag:yaml.org,2002:python/name:pymdownx.superfences.fence_code_format"
+              ]
             }
             ```
 
+            `yaml.customTags` part will be needed if you want to use and configure [Emojis](reference/icons-emojis.md).
```

Comment:
Could you please move this into a [code annotation](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#code-annotations) on line 66 in the source file?

Reply:
I think I moved it into code annotation